test/functional/page_objects/visualize_page.js

```diff
@@ -248,17 +248,28 @@ export function VisualizePageProvider({ getService, getPageObjects }) {
 
     async setComboBox(comboBoxSelector, value) {
       const comboBox = await testSubjects.find(comboBoxSelector);
-      await this.setComboBoxElement(comboBox, value);
+      await this.setComboBoxValue(comboBox, value);
+      await this.closeComboBoxOptionsList(comboBoxSelector);
       await PageObjects.common.sleep(1000);
     }
 
-    async setComboBoxElement(element, value) {
-      const input = await element.findByTagName('input');
+    async setComboBoxElement(comboBox, value) {
```

Comment:
I had to refactor out common functionality between `setComboBox` and `setComboBoxElement` into `setComboBoxValue` and then remove the dependency upon `setComboBoxElement` from `setComboBox` so that we could pass down the test subject selector to the other helper methods.